public/js/main.js

```diff
@@ -200,3 +200,90 @@ async function updateAddress() {
   }
   
 }
+
+// Upload Photo from barista profile to Cloudinary 
+let signData = new Object()
+let cloudinary_url = new String()

```

Comment:
Prefer `camelCase` in JS variables.

Reply:
Having two functions depend on this variable makes them dependent on each other. Ie, what happens if `uploadProfilePhoto` was called before `fetchSignature`?

Maybe you've architectured your UI in a way that would prevent that, but it makes your code less resilient to refactoring later. Why not just call `fetch('/barista/signuploadform', { ...` in the `uploadProfilePhoto` function? I might be missing something though :)